src/com/elixr/training/Main.java

```diff
@@ -3,20 +3,24 @@
 import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.StringTokenizer;
 import java.lang.String;
+import java.sql.*;
+import com.elixrdb.training.JdbcConnection;
 
 public class Main {
-    static  final String TXTFILE = ".txt" ;
-    static final String JSONFILE = ".json";
-    public static void main(String[] args) {
+    static JdbcConnection database = new JdbcConnection();
+    static int count = 0;
+    static String inputFilePath;
+    static String searchWord ;
 
+    public static void main(String[] args) throws SQLException, InterruptedException {
```

Comment:
You are not logging failures. As per the PoC document
`All runs of the application must be logged to a MySQL database, table name ‘audit’. The table should have the following columns, path to the file, the word searched, the date time of the search, result (success or error), word count(numerical field, can be 0 if error occurred), error message(would be empty if no error occurred)`

You should be logging failures and errors also in the database

Reply:
added new column for error message and have passed message